src/LinAlg/hiopVectorPar.cpp

```diff
@@ -804,4 +804,19 @@ void hiopVectorPar::print(FILE* file, const char* msg/*=NULL*/, int max_elems/*=
   }
 }
 
+
+long long hiopVectorPar::numOfElemsLessThan(const double &val) const
```

Comment:
hiopVectorPar is supposed to support distributed (MPI) vectors. We need to either implement full functionality for the MPI case (only need to do MPI_AllReduce) or assert(false) if the method(s) is called for a distributed vector (assert(n_local_==n_global_);